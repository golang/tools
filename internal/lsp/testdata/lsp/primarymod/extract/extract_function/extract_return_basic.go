package extract

func _() bool {
	x := 1
	if x == 0 { //@mark(s0, "if")
		return true
	} //@mark(e0, "}")
	return false
	//@extractfunc(s0, e0)
}
