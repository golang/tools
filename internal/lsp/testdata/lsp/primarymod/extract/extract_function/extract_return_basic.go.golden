-- functionextraction_extract_return_basic_5_2 --
package extract

func _() bool {
	x := 1
	cond0, ret0 := x0(x)
	if cond0 {
		return ret0
	} //@mark(e0, "}")
	return false
	//@extractfunc(s0, e0)
}

func x0(x int) (bool, bool) {
	if x == 0 {
		return true, true
	}
	return false, false
}

